From ab21f27d000ca3ffd6e25b9bfe0d8de25b9d5c67 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Mon, 24 Oct 2011 16:22:36 +0800
Subject: [PATCH 1/2] P1022ds:DTS:Add pixis support

Add pixis node to dts for P1022ds board.

Signed-off-by: Tang Yuantian <b29983@freescale.com>

[Extracted from vendor drop QorIQ-NONDPAA-SDK-V1-20110429_ltib.iso
linux-2.6.35-qoriq-P1022ds-DTS-Add-pixis-support.patch
Add pixis node to 36bit dts for P1022ds board.]

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/powerpc/boot/dts/p1022ds.dts       |    4 ++++
 arch/powerpc/boot/dts/p1022ds_36bit.dts |    4 ++++
 2 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/p1022ds.dts b/arch/powerpc/boot/dts/p1022ds.dts
index d47da03..34cbc02 100644
--- a/arch/powerpc/boot/dts/p1022ds.dts
+++ b/arch/powerpc/boot/dts/p1022ds.dts
@@ -142,6 +142,10 @@
 				label = "reserved-nand";
 			};
 		};
+		board-control@3,0 {
+			compatible = "fsl,p1022ds-fpga-pixis";
+			reg = <0x3 0x0 0x8000>;
+		};
 	};
 
 	soc@ffe00000 {
diff --git a/arch/powerpc/boot/dts/p1022ds_36bit.dts b/arch/powerpc/boot/dts/p1022ds_36bit.dts
index 0c6f0f0..c8d0bc3 100644
--- a/arch/powerpc/boot/dts/p1022ds_36bit.dts
+++ b/arch/powerpc/boot/dts/p1022ds_36bit.dts
@@ -149,6 +149,10 @@
 				label = "reserved-nand";
 			};
 		};
+		board-control@3,0 {
+			compatible = "fsl,p1022ds-fpga-pixis";
+			reg = <0x3 0x0 0x8000>;
+		};
 	};
 
 	soc@fffe00000 {
-- 
1.7.0.4

